common/src/main/java/com/wynntils/models/worlds/bossbars/InfoBar.java

```diff
@@ -33,17 +33,17 @@ public InfoBar() {
     }
 
     @Override
-    public void onUpdateName(Matcher matcher) {
-        if (matcher.pattern().equals(BOMB_INFO_PATTERN)) {
-            BombType bombType = BombType.fromString(matcher.group("bomb"));
+    public void onUpdateName(Matcher match) {
```

Comment:
If this is a cleanup PR, why rename random stuff? we called Matchers matcher like a 1000 places in the codebase

Reply:
Intellij got the name from the parent class (I think bar)

Reply:
i just changed one of them to match. the others where there before
![image](https://github.com/Wynntils/Artemis/assets/66840021/63c09053-7367-4491-bc8f-1ca7255aa9f8)

Reply:
And, once again, don't do this. Please revert the rename changes.